embrace-android-sdk/src/main/java/io/embrace/android/embracesdk/worker/WorkerThreadModule.kt

```diff
@@ -35,7 +35,7 @@ internal enum class ExecutorName(internal val threadName: String) {
     CACHED_SESSIONS("cached-sessions"),
     SEND_SESSIONS("send-sessions"),
     DELIVERY_CACHE("delivery-cache"),
-    API_RETRY("api-retry"),
+    NETWORK_REQUEST("api-retry"),
```

Comment:
is there any reason for not renaming "api-retry" to "network_request"?

Reply:
Yeah I'd definitely change this as the thread name shows up in traces

Reply:
No reason. I've renamed it